### PR TITLE
PHP7 features, cleanup

### LINKED ISSUE #33: Utilize PHP 7.1 features

### PR DESCRIPTION
**Pull request by szymach**
After #34, resolves #33

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

File: Form/EventListener/SortableCollectionListener.php [hunk 1]
```diff
@@ -70,26 +71,17 @@ private function getEventDataKeys(FormEvent $event)
...
+    private function getRememberedItemOrder(FormEvent $event): ?array
     {
         $formId = $this->getEventFormId($event);
 
-        if (isset($this->itemOrder[$formId])) {
-            return $this->itemOrder[$formId];
-        } else {
-            return null;
-        }
+        return isset($this->itemOrder[$formId])
```
Comment (chives) [inline Form/EventListener/SortableCollectionListener.php]: why not `return $this->itemOrder[$formId] ?? null` ?

Comment (szymach) [reply Form/EventListener/SortableCollectionListener.php]: I was not aware it silences nonexistent key notices.

**Comment by szymach** [comment]
@fsi-open

**Pull request merged**